frontend/src/views/UserDetails/index.vue

```diff
@@ -28,6 +32,19 @@ const navigateBack = () => {
     });
 };
 
+/**
+ * Get the FamApplicationDto with the applicationId,
+ * it is used to display the app name in subtitle.
+ */
+const adminUserAccessQuery = useQuery({
+    queryKey: ["admin-user-access"],
+    queryFn: () =>
+        AdminMgmtApiService.adminUserAccessesApi
+            .adminUserAccessPrivilege()
+            .then((res) => res.data),
+    select: (data) => getApplicationById(Number(applicationId), data),
+});
```

Comment:
No change, Just wonder, do we fetch this every time since I see it at several places or do we intend to fetch every time? Or is this cached?

Reply:
if there's a cached value it will be using the cached data, if there isn't a cached value, e.g. after a refresh, then the data will be automatically fetched again. 

However, in manage permission view, this data is refetched on every mount with 
```javascript
refetchOnMount: true,
```

This refetch (when there's a cached data already) will not trigger the skeleton of the dropdown to show, since it's the skeleton will only show when `query.isLoading === true`, there's a small difference between `query.isFetching` and `query.isLoading`, loading indicates that this the first data load (no cached value), fetching is true whenever the data is fetching, meaning if loading is true then fetching is true, but fetching is true doesn't mean loading is true. This allows us to silently refresh the data